python/pyspark/mllib/fpm.py

```diff
@@ -94,12 +100,9 @@ def train(cls, data, minSupport=0.3, numPartitions=-1):
         model = callMLlibFunc("trainFPGrowthModel", data, float(minSupport), int(numPartitions))
         return FPGrowthModel(model)
 
-    class FreqItemset(namedtuple("FreqItemset", ["items", "freq"])):
-        """
-        Represents an (items, freq) tuple.
+    # Backward-compatible alias.
+    FreqItemset = FreqItemset
```

Comment:
I think this line should be removed either directly, or following a deprecation cycle.